website/docs/r/dns_managed_zone.html.markdown

```diff
@@ -277,7 +277,7 @@ The following arguments are supported:
 * `project` - (Optional) The ID of the project in which the resource belongs.
     If it is not provided, the provider project is used.
 
-
```

Comment:
The deleted empty line breaks formatting here:
https://www.terraform.io/docs/providers/google/r/dns_managed_zone.html#force_destroy